lib/definition_table_generator.rb

```diff
@@ -24,6 +28,10 @@ def terminology_table(content_key)
     render_table(content_key, 'terminology', 'terminology_table')
   end
 
+  def patch_definition_table(content_key)
+    render_table(content_key, "#{@action}", 'patch_definition_table', 'patch')
```

Comment:
This should just be: `render_table(content_key, 'patch_operation', 'patch_definition_table')`

Reply:
Fixed. https://github.com/cerner/fhir.cerner.com/pull/240/commits/95e5482ad098fdd0ddd84bb68aded37c3a9b6650#diff-73e703ab5c2efa2a629ba1622a8cd56dR32